content/src/main/content/jcr_root/apps/acs-commons/components/content/long-form-text/dialog.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+        jcr:primaryType="cq:Dialog"
+        activeTab="0"
+        title="Long Form Text"
+        xtype="tabpanel">
+    <items jcr:primaryType="cq:WidgetCollection">
+        <tab1
+                jcr:primaryType="cq:Widget"
+                title="Text"
+                xtype="panel">
+            <items jcr:primaryType="cq:WidgetCollection">
+                <long-form-text
```

Comment:
Is there any way (or reason?) to get this to use the generic text configuration feature? Otherwise, users will need to override this dialog whenever they want different RTE options.